link/platform.go

```diff
@@ -0,0 +1,19 @@
+package link
+
+import (
+	"fmt"
+	"runtime"
+)
+
+func platformPrefix(symbol string) string {
+
+	prefix := runtime.GOARCH
+	switch prefix {
+	case "i386":
```

Comment:
This is not a valid GOARCH value. All possible values are here: https://github.com/golang/go/blob/master/src/go/build/syslist.go.

Reply:
True, my eyes misled me on this. From what I recall I checked this gist -> https://gist.github.com/asukakenji/f15ba7e588ac42795f421b48b8aede63#a-list-of-valid-goarch-values